creativecloud/features/firefly/firefly-interactive.js

```diff
@@ -93,34 +95,39 @@ export default async function setInteractiveFirefly(el) {
   const ttiDetail = {};
   const genfDetail = {};
   const teDetail = {};
-  const allSelections = [...media.querySelectorAll('p:not(:empty)')].filter((p) => p.innerText.trim().toLowerCase().includes('interactive-selectors'));
+  const fireflyDomain = [
+    'firefly.adobe.com',
```

Comment:
Can we remove this as an array and keep it as a constant string if that is the only thing required?

Reply:
CodeQl contains if we keep it as string